src/entities/TieredSto.ts

```diff
@@ -166,23 +234,23 @@ export class TieredSto extends Sto<Params> {
-   * @param endDate date when the STO should end
-   * @param tiers tier information
-   * @param tiers[].tokensOnSale amount of tokens to be sold on that tier
-   * @param tiers[].price price of each token on that tier
-   * @param tiers[].tokensWithDiscount amount of tokens to be sold on that tier at a discount if paid in POLY (must be less than tokensOnSale, defaults to 0)
-   * @param tiers[].discountedPrice price of discounted tokens on that tier (defaults to 0)
-   * @param nonAccreditedInvestmentLimit maximum investment for non-accredited investors
-   * @param minimumInvestment minimum investment amount
-   * @param fundraiseCurrencies array of currencies in which the funds will be raised (ETH, POLY, StableCoin)
-   * @param raisedFundsWallet wallet address that will receive the funds that are being raised
-   * @param unsoldTokensWallet wallet address that will receive unsold tokens when the end date is reached
-   * @param stableCoinAddresses addresses of supported stablecoins
-   * @param customCurrency custom currency data. Allows the STO to raise funds pegged to a different currency. Optional, defaults to USD
-   * @param customCurrency.currencySymbol symbol of the custom currency (USD, CAD, EUR, etc. Default is USD)
-   * @param customCurrency.ethOracleAddress address of the oracle that states the price of ETH in the custom currency. Only required if raising funds in ETH
-   * @param customCurrency.polyOracleAddress address of the oracle that states the price of POLY in the custom currency. Only required if raising funds in POLY
+   * @param args.startDate - date when the STO should start
+   * @param args.endDate - date when the STO should end
+   * @param args.tiers - tier information
+   * @param args.tiers[].tokensOnSale - amount of tokens to be sold on that tier
+   * @param args.tiers[].price - price of each token on that tier
+   * @param args.tiers[].tokensWithDiscount - amount of tokens to be sold on that tier at a discount if paid in POLY (must be less than tokensOnSale, defaults to 0)
+   * @param args.tiers[].discountedPrice - price of discounted tokens on that tier (defaults to 0)
+   * @param args.nonAccreditedInvestmentLimit - maximum investment for non-accredited investors
+   * @param args.minimumInvestment - minimum investment amount
+   * @param args.fundraiseCurrencies - array of currencies in which the funds will be raised (ETH, POLY, StableCoin)
+   * @param args.raisedFundsWallet - wallet address that will receive the funds that are being raised
+   * @param args.unsoldTokensWallet - wallet address that will receive unsold tokens when the end date is reached
+   * @param args.stableCoinAddresses - addresses of supported stablecoins
+   * @param args.customCurrency - custom currency data. Allows the STO to raise funds pegged to a different currency. Optional, defaults to USD
+   * @param args.customCurrency.currencySymbol - symbol of the custom currency (USD, CAD, EUR, etc. Default is USD)
```

Comment:
From the guide: "Arg properties which are complex interfaces defined elsewhere should not be broken down into their own properties (the autogen tool provides a link to the corresponding interface)"

This applies to all the properties within args.customCurrency